geode-core/src/test/java/org/apache/geode/internal/net/SocketCreatorJUnitTest.java

```diff
@@ -98,6 +102,30 @@ private void testBindExceptionMessageFormatting(InetAddress inetAddress) throws
     }
   }
 
+  @Test
+  public void configureSSLEngine() {
+    SSLConfig config = new SSLConfig.Builder().setCiphers("someCipher").setEnabled(true)
```

Comment:
What about a test for `"any"`?

Reply:
Yes!  I'm having lots of "fun" with "any" this week and that's an excellent idea.